src/pages/Booking/Seats/SeatsButton.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+function SeatsButton({
+  item,
+  selectedSeat,
+  addSeats,
+  removeSeats,
+  peopleTotalSum,
+  toggle,
+}) {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const checkedSeat = e => {
+    if (peopleTotalSum > selectedSeat.length) {
+      setIsChecked(prev => !prev);
+    }
+    if (peopleTotalSum <= selectedSeat.length) {
+      setIsChecked(prev => prev);
+    }
+  };
+
+  useEffect(() => {
+    setIsChecked(false);
+  }, [toggle]);
```

Comment:
```suggestion
const isChecked = selectedSeat.includes(
    String(item.seatStatus[0].timeTableSeatId)
  );
```

해당 부분도 부모에서 전달하는 Props 값으로 계산 가능한 값이기 때문에
state를 만들지 않아도 됩니다!